src/Uno.UWP/Storage/Pickers/FilePickerHelper.Android.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Android.App;
+using Android.Content;
+
+namespace Windows.Storage.Pickers
+{
+	public static class FilePickerHelper
+	{
+		public static IDisposable RegisterOnBeforeStartActivity(FileOpenPicker picker, Intent intent)
```

Comment:
Thanks for your PR!

A few things:
- What's the use of the disposable return in this context?
- This change allows for overriding the intent completely, which _may_ create an incompatible intent for the actions performed. What's the use case behind providing a new intent?
- Can you add some documentation of the suggested use of this specific feature?